### PR TITLE
Remove wallet from base node

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
Removed the internal base node wallet from the base node application, as it is now deprecated.

_**Note:** Dependent on #2709; must be merged after that one._

## Motivation and Context
See above

## How Has This Been Tested?
- Unit tests
- System-level tests:
  - base node sync from scratch
  - base node, stand-alone wallet, mining node

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [X] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [X] I ran `cargo-fmt --all` before pushing.
* [X] I ran `cargo test` successfully before submitting my PR.
* [X] I have squashed my commits into a single commit.
* [X] My change requires a change to the documentation.
* [X] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by delta1** [comment]
needs a cargo fmt

**Comment by delta1** [comment]
it's awesome to see these big red PRs, minus 2k lines! 👏

File: applications/tari_base_node/src/command_handler.rs [hunk 1]
```diff
@@ -1243,625 +930,21 @@ impl CommandHandler {
...
-            let code = QrCode::new(qr_link).unwrap();
-            let image = code
-                .render::<unicode::Dense1x2>()
-                .dark_color(unicode::Dense1x2::Dark)
-                .light_color(unicode::Dense1x2::Light)
-                .build();
-            println!("{}", image);
-            println!();
-        }
         println!("======== Base Node ==========");
```
Comment (SWvheerden) [inline applications/tari_base_node/src/command_handler.rs]: We can remove this list as well. There is only going to be the basenode in the whoami

File: applications/tari_base_node/src/parser.rs [hunk 2]
```diff
@@ -103,12 +88,6 @@ pub struct Parser {
     command_handler: Arc<CommandHandler>,
 }
 
-const MAKE_IT_RAIN_USAGE: &str = "\nmake-it-rain [Txs/s] [duration (s)] [start amount (uT)] [increment (uT)/Tx] \
-                                  [\"start time (UTC)\" / 'now' for immediate start] [public key or emoji id to send \
-                                  to] [message]\n";
```
Comment (SWvheerden) [inline applications/tari_base_node/src/parser.rs:108]: We need to add this the console_wallet. If we remove this, we lose this piece of documentation. The console wallet only has the command listed.

Comment (SWvheerden) [reply applications/tari_base_node/src/parser.rs:108]: We should probably add one for coin-split as well. We are deleting its only documentation as well.

File: common/src/configuration/global.rs [hunk 3]
```diff
@@ -77,9 +77,7 @@ pub struct GlobalConfig {
     pub base_node_tor_identity_file: PathBuf,
     pub wallet_db_file: PathBuf,
     pub console_wallet_db_file: PathBuf,
-    pub wallet_identity_file: PathBuf,
     pub console_wallet_identity_file: PathBuf,
-    pub wallet_tor_identity_file: PathBuf,
     pub console_wallet_tor_identity_file: PathBuf,
```
Comment (SWvheerden) [inline common/src/configuration/global.rs:81]: We should probably look at removing these as well. The wallet is not supposed to use them anymore. Its saves this data inside of the database.

**Review by SWvheerden — COMMENTED** [review]
Looks good, we should just ensure we don't lose all the current documentation. I left comments on those. We can also remove some of the other console_wallet id files They should not be used.

File: applications/tari_base_node/src/parser.rs (hunk 2 above)
Comment (delta1) [reply applications/tari_base_node/src/parser.rs:108]: Both of those are already in the console wallet.

File: common/src/configuration/global.rs (hunk 3 above)
Comment (hansieodendaal) [reply common/src/configuration/global.rs:81]: Agreed, they should be removed, but I think its for another PR, has to do with the console wallet.

File: applications/tari_base_node/src/parser.rs (hunk 2 above)
Comment (hansieodendaal) [reply applications/tari_base_node/src/parser.rs:108]: Agreed; another PR.

**Pull request merged**